### PR TITLE
ESLint + prettier

### LINKED ISSUE #19: Get ESLint/Prettier combination

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #19 

To be honest I've not really used prettier much, so I don't even know if I'm supposed to call `yarn lint` or `yarn pretty` or both.

This is already way too over-engineered for this scale of project

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+	"useTabs": true,
+	"printWidth": 120,
+	"bracketSpacing": false
```
Comment (adirar111) [inline .prettierrc.json:4]: ah, so you chose violence

**Comment by adirar111** [comment]
> To be honest I've not really used prettier much, so I don't even know if I'm supposed to call yarn lint or yarn pretty or both.

wdym, this is the 310 setup :skull:

> This is already way too over-engineered for this scale of project

best extension engineering in the game, bb

**Review by adirar111 — APPROVED** [review]
idk either, but this seems legit

**Comment by braxtonhall** [comment]
> > To be honest I've not really used prettier much, so I don't even know if I'm supposed to call yarn lint or yarn pretty or both.
> 
> wdym, this is the 310 setup 💀


yeah I know I'm the one who set it up!

**Pull request merged**